webClient/src/app/app.component.ts

```diff
@@ -45,26 +45,37 @@ export class AppComponent {
   }
 
   ngOnInit() {
+    let activeZluxEditors : number = +window.localStorage.getItem("activeZluxEditors")
+    activeZluxEditors > 0 ? activeZluxEditors++ : activeZluxEditors = 1 
+    window.localStorage.setItem("activeZluxEditors",activeZluxEditors.toString())
+    window.addEventListener("beforeunload", () => { 
+      activeZluxEditors = +window.localStorage.getItem("activeZluxEditors")
```

Comment:
also, there is a difference between localStorage and sessionStorage.
localStorage may be saved to disk, but I don't think sessionStorage ever is. Session storage is removed when the browser is closed, local storage remains. I don't think there is value of keeping this after the browser is closed, so session storage seems better.